addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SqlInjectionOracleTimingScanRule.java

```diff
@@ -74,7 +74,6 @@ public class SqlInjectionOracleTimingScanRule extends AbstractAppParamPlugin
             "SELECT  UTL_INADDR.get_host_name('10.0.0.1') from dual union SELECT  UTL_INADDR.get_host_name('10.0.0.2') from dual union SELECT  UTL_INADDR.get_host_name('10.0.0.3') from dual union SELECT  UTL_INADDR.get_host_name('10.0.0.4') from dual union SELECT  UTL_INADDR.get_host_name('10.0.0.5') from dual";
```

Comment:
Maybe I should leave this for another time/another PR but I don't think there's any need for the double spaces here before UTL_INADDR

Reply:
More so if it's going to be replaced(?).